torch/quantization/quantize.py

```diff
@@ -42,8 +42,7 @@ def _propagate_qconfig_helper(module, qconfig_dict, white_list=None,
     module_qconfig = qconfig_dict.get(prefix, module_qconfig)
     module_qconfig = getattr(module, 'qconfig', module_qconfig)
 
-    if type(module) in white_list:
-        module.qconfig = module_qconfig
+    module.qconfig = module_qconfig
```

Comment:
How do we make sure only the ones that are allowed are getting the qconfig?

Reply:
yeah, I realized that, just updated the diff. Ideally we don't have people using this "white_list" API, otherwise we'll need to make changes to callsite.

Reply:
The numeric suite tests might be impacted by this change, please check if they are passing prior to landing.

Reply:
Also, what is the motivation of this change? Why do we need a qconfig for all modules, even if they are not observed?